seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/AbstractSeleniumJSRunner.scala

```diff
@@ -3,11 +3,17 @@ package org.scalajs.jsenv.selenium
 import org.scalajs.core.tools.io.{MemVirtualJSFile, VirtualJSFile}
 import org.scalajs.core.tools.jsdep.ResolvedJSDependency
 import org.scalajs.core.tools.logging.Logger
-import org.scalajs.jsenv.JSConsole
+import org.scalajs.jsenv.{JSConsole, VirtualFileMaterializer}
 
 abstract class AbstractSeleniumJSRunner(browserProvider: SeleniumBrowser,
     libs: Seq[ResolvedJSDependency], code: VirtualJSFile, materializer: FileMaterializer) {
 
+  @deprecated("Added FileMaterializer to the parameters. Falling back to DefaultFileMaterializer.", "0.1.2")
```

Comment:
Consider the message

    Use the overload with an explicit FileMaterializer.

which is the "format" of deprecation messages that we typically use.